x-pack/plugins/security_solution/public/common/components/ml/tables/translations.ts

```diff
@@ -65,3 +65,35 @@ export const NETWORK_NAME = i18n.translate('xpack.securitySolution.ml.table.netw
 export const TIME_STAMP = i18n.translate('xpack.securitySolution.ml.table.timestampTitle', {
   defaultMessage: 'Timestamp',
 });
+
+export const JOB_ID = i18n.translate('xpack.securitySolution.ml.table.jobIdFilter', {
+  defaultMessage: 'Job',
+});
+
+export const INTERVAL_TOOLTIP = i18n.translate('xpack.securitySolution.ml.table.intervalTooltip', {
```

Comment:
I don't quite understand the `auto` option in the interval 😅 maybe we could explain it here. Has this text been agreed with @paulewing?

Reply:
Okay, it is good to have consistency, but I still don't understand what `auto` does 🤷‍♂️